pandas/core/generic.py

```diff
@@ -3156,6 +3156,13 @@ def copy(self, deep=True):
         data = self._data.copy(deep=deep)
         return self._constructor(data).__finalize__(self)
 
+    __copy__ = copy
```

Comment:
Not sure I understand -- `def copy` is defined immediately above, and the PR adds the `__copy__ = copy` alias.  Are you saying that this won't work unless I also do `def __copy__`?